src/frontend/components/Footer/Footer.css

```diff
@@ -0,0 +1,25 @@
+.footer {
+    background-color: var(--black-color);
+    padding: 2rem 0rem;
+    text-align: center;
+    color: white;
+    margin-top: 5rem;
+}
+
+.footer__links{
+    align-items: center;
+    justify-content: center;
+}
+
+.footer__socials {
+    color: var(--white-color);
+    padding: 2rem;
+    font-size: 4rem;
+}
+
+hr {
+    margin: 2rem;
+    border: 0;
+    height: 4px;
```

Comment:
For height use vh unit

Reply:
its just an horionzal line so it doesnt need a much of height